example/docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+version: "2.4"
+services:
+  borgmatic:
+    container_name: borgmatic
+
+    image: modem7/borgmatic-docker
+    #build: ../base
+
+    volumes:
+        # -------------
+        # Source and destination directories
+
+        # Mount the directories you want to back up under /mnt/source.
+        # We're mounting them read-only for a little extra safety.
+        - /home:/mnt/source/home:ro
+        - /etc:/mnt/source/etc:ro
+
+        # If you're backing up to a local backup destination, mount it here.
+        # Remove this if you're doing remote backups only.
+        - /mnt/backup:/mnt/borg-repository
+
+        # -------------
+        # Config files that you write
+
+        # Borgmatic and crontab config files will go here.
+        - ./borgmatic.d:/etc/borgmatic.d/
+
+        # If you're using a remote repository, you'll need an SSH key.
+        # Uncomment this if you want to use root's ssh key.
+        #- ~/.ssh:/root/.ssh
+
+        # If you want a separate SSH key just for borg, uncomment this instead, then `ssh-keygen -f ssh/id_rsa`.
+        #- ./ssh:/root/.ssh
+
```

Comment:
What about the Borgmatic state files?

`./.borgmatic:/root/.borgmatic`

Reply:
The [README](https://github.com/borgmatic-collective/docker-borgmatic/tree/master/base) says "this is now redundant and has been deprecated, please remove this from your configs", so I did. I checked my running install and there is a `checks` directory with a single 64-character-hex-hash subdirectory with a single 64-character-hex-hash-named empty file; I'm not sure what's going on there.

If you say this is important I'll put it back, but if so, the README should also be updated.

Reply:
I missed the [README](https://github.com/borgmatic-collective/docker-borgmatic/tree/master/base) update. You are right. It is not needed anymore.